<launch>
  <node pkg="video_writer_c" name="video_writer_c" type="video_writer_c_node" output="screen">
  <param name="show_video_flag" type="bool" value="true"/>
  <param name="save_result_video_flag" type="bool" value="true"/>
  <param name="loop_rate" type="double" value="30"/>
  <param name="cam_num" type="int" value="1"/>
  <param name="result_video_location" type="string" value="/home/kevin/workspace/IARC/iarc2018.7.16/src/video_writer_c/result_video/result.avi"/>

  </node>
</launch>
